tools/lsp/README.md

```diff
@@ -118,3 +118,28 @@ exists under the `language` server section:
   }
 }
 ```
+
+## Sublime Text
+
+To setup the lsp:
+1. Install the sixtyfps-lsp binary with `cargo install` as explained in the *Generic Usage* section above.
+2. Using Package Control in Sublime Text, install the LSP package (sublimelsp/lsp)
```

Comment:
Should that be a link?

Reply:
You have to select it from a menu:

<img width="465" alt="image" src="https://user-images.githubusercontent.com/839935/116820196-fe3e0c00-ab73-11eb-87e5-ac7896ec2958.png">